src/components/profile-update/profile-update.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Form, Button, Card, CardGroup, Container, Col, Row } from 'react-bootstrap';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+import React, { useState } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+
+
+
+export function ProfileUpdate(props) {
+
+
+    console.log("props: ", props);
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [email, setEmail] = useState('');
+    const [birthday, setBirthday] = useState('');
+
+    const handleUpdateUser = (e) => {
+        e.preventDefault();
+        /* Send a request to the server for authentication */
+        axios.put('https://morning-badlands-52426.herokuapp.com/users', {
```

Comment:
This API url for an update API call does not look right. But if it works alright the way it is, we cal let it slide. Does it work as expected when you test the API in postman?

Reply:
POSTMAN: I first login and get the token after successful login, 
then go to PUT request update user by name and paste the token in Authorisation, 
then change the URL to https://morning-badlands-52426.herokuapp.com/users/<current logged in user name>, 
update the array in Body JSON format:
{
    "Username": "updatedUser",
    "Password": "updatedUser", (password isn't hashed)
    "Email": "updatedUser@gmail.com",
    "Birthday": "2001-01-01"
}
I got the response STATUS 200 OK and new array with updated details, the user _id stays the same.
I also can see the updated user in Mongo DB with password not being hashed, 
so cannot login via the app as the pass is not being recognized. But I think that's ok?

Reply:
Alright, so the url you are making a request to in the react application `https://morning-badlands-52426.herokuapp.com/users` is missing the `/username` I suppose.